Gamma_Analysis.py

```diff
@@ -43,13 +40,16 @@ def absolute_efficiency(energy, coeffs=EFFICIENCY_CAL_COEFFS):
     return efficiency
 
 
-def emission_rate(net_area, efficiency, livetime):
+def emission_rate(net_area, livetime):
```

Comment:
Yes, this is what I suggested doing in my email. However now that I'm looking at it, there are some problems.
* This can't be called `emission_rate` anymore, it is just a peak countrate
* The `isotope_activity` function now gets a list of peak countrates instead of emission rates, and the weighted average in this doesn't make sense anymore
There is still a better/simpler way to do it without using the efficiency, but it would require even more restructuring, so for now let's leave it be. Maybe later.

So put `emission_rate` and the relevant calls in `analyze_isotope` back the way they were. Thanks.